app/kudos/kudos.yaml

```diff
@@ -5118,3 +5118,11 @@
   - Friends
   - Love
   image: 3box_butterfly.svg
+- name: DAI
```

Comment:
i dont know why this is on my PR.  it already exists on master.

Reply:
😓  Is github UI messed up again